controllers/provisioners/eks/helpers.go

```diff
@@ -159,6 +159,15 @@ mount
 echo "{{ .Device}}    {{ .Mount }}    {{ .FileSystem | ToLower }}    defaults    0    2" >> /etc/fstab
 {{- end}}
 {{- end}}
+if [[ $(type -P $(which aws)) ]] && [[ $(type -P $(which jq)) ]] ; then
+	INSTANCE_ID=$(curl http://169.254.169.254/latest/meta-data/instance-id)
```

Comment:
Can we default to using IMDS v2? Some organizations might have a security policy around only allowing IMDS v2 to be used. https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-service.html